CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+First of all, thank you for contributing. It’s appreciated.
+
+# To submit a pull request
+
+1. Open a GitHub issue before doing significant amount of work.
+2. Clone the repo. If it was already cloned, then git pull to get the latest from master.
+4. Run `npm install`.
+5. Write code.
+6. Run `npm test` to lint and test. Don’t commit before fixing all errors and warnings.
+7. Commit using `npm run commit` and follow the CLI instructions.
+8. Make a pull request.
+
+# To release new versions
+
+1. Check that you have npm publishing rights before anything else.
+2. Run `npm run check-release`.
```

Comment:
There is now, i saw we were using on other cycle repos, nice little northbrook feature!